src/app/components/Hits/Hits.jsx

```diff
@@ -68,6 +68,19 @@ class Hits extends React.Component {
     ));
   }
 
+  clearResults() {
+    Actions.updateSpinner(true);
+    Actions.updateSearchKeywords('');
+    Actions.updateSelectedFacets({});
+    ajaxCall(`/api`, (response) => {
+      Actions.updateSearchResults(response.data.searchResults);
+      Actions.updateFacets('');
```

Comment:
This should be `response.data.facets` I think. Otherwise the sidebar will be empty instead of displaying the default facets from the API.